tests/streaming/agent/test_utils.py

```diff
@@ -25,6 +24,11 @@ class StreamOpenAIResponseTestCase(BaseModel):
     expected_sentences: List[str]
 
 
+class StreamOpenAIResponseFunctionTestCase(BaseModel):
```

Comment:
can you merge these into a single test case class? with functions = True

if we're testing the same method seems cleaner to just have a single class and a single for loop